tests/test_aws.py

```diff
@@ -61,6 +61,23 @@ class ArnResolverTest(BaseTest):
         ('arn:aws:autoscaling:region:account-id:autoScalingGroup:groupid:autoScalingGroupName/groupfriendlyname', 'asg') # NOQA
     ]
 
+    def test_arn_resolve_resources(self):
+        arns = [
+            'arn:aws:sqs:us-east-1:644160558196:origin-dev',
+            'arn:aws:lambda:us-east-1:644160558196:function:custodian-sg-modified',
+            'arn:aws:lambda:us-east-1:644160558196:function:custodian-s3-tag-creator:$LATEST',
```

Comment:
we want to add tests here for valid/invalid arns across categories that may apply for an event hub rule target

Reply:
this can happen in some follow up branches directly targeted to #6199